src/dpx.imageio/dpxinput.cpp

```diff
@@ -43,6 +48,8 @@ class DPXInput final : public ImageInput {
     std::vector<unsigned char> m_userBuf;
     bool m_rawcolor;
     std::vector<unsigned char> m_decodebuf;  // temporary decode buffer
+    int64_t m_io_offset       = 0;
+    Filesystem::IOProxy* m_io = nullptr;
```

Comment:
I believe you also want to add

    virtual bool set_ioproxy(Filesystem::IOProxy* ioproxy) override

to this class

Reply:
Actually, set_ioproxy may only be a thing for 2.2, not 2.1.

If you're liking ioproxies, you should really try to get to 2.2 as soon as you can. There were several interesting enhancements.

Reply:
I've seen some in the latest docs that it's easier create ImageInput-s which read from memory. But I'm somewhat dependent on Vcpkg and especially on time. Maybe we'll switch to 2.2 when a newer version of OpenEXR is available on Vcpgkg -- I don't think I'll get the change to learn how to modify and modify its "ports".